apps/site/src/components/component-blocks/components/availability-content/components/table-of-availability/table-of-availability.component.tsx

```diff
@@ -41,7 +41,7 @@ export function TableOfAvailability({
   const styles = TableOfAvailabilityStyles({});
 
   const hasAlternativeNames = platforms.some(platform => platform.alternative);
-  const cellWidth = 200;
+  const cellWidth = '10%';
```

Comment:
Update to `const cellWidth = hasAlternativeNames ? '33%' : '50%';` 
i.e. if has 3 cols divide space into thirds otherwise for standard 2 col divide space into 50%